app/deployments/tests/test_models.py

```diff
@@ -1,4 +1,6 @@
+from django.core.management import call_command
 from django.test import TestCase
+import pytest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'pytest'](https://app.codacy.com/manual/gmri/buoy_barn/pullRequest?prid=4987082)